src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -81,56 +82,110 @@ LoadFeeTrack::lowerLocalFee ()
 
 //------------------------------------------------------------------------------
 
-// NIKB TODO: Once we get C++17, we can replace lowestTerms
-//            with this:
-//
-// template <class T1, class T2,
-//     class = std::enable_if_t<
-//         std::is_integral_v<T1> &&
-//         std::is_integral_v<T2>>
-// >
-// void lowestTerms(T1& a,  T2& b)
-// {
-//     if (auto const gcd = std::gcd(a, b))
-//     {
-//         a /= gcd;
-//         b /= gcd;
-//     }
-// }
+// EHENNIS TODO: Once we get C++17, we can replace gcd with std::gcd
+
+template <class T>
+std::enable_if_t<std::is_unsigned_v<T>, T>
+abs(T t)
+{
+    return t;
+}
+
+template <class T>
+std::enable_if_t<std::is_signed_v<T>, T>
+abs(T t)
```

Comment:
Given how `abs` is used, I might have both functions return `std::uint64_t`. For the signed version, I'd cast to `int64` before taking the abs (E.g. as written we'll get the wrong answer for `numeric_limits<std::int8_t>::min()`).

Reply:
My intention was to have the same interface as `std::abs` without writing a bunch of overloads. But I've fixed it now.